src/ca/team2706/vision/trackerboxreloaded/Main.java

```diff
@@ -306,34 +411,49 @@ public static void main(String[] args) {
         // Main video processing loop
         while (true) {
             if (useCamera) {
+            	//Read the frame from the camera, if it fails try again
                 if (!camera.read(frame)) {
                     System.err.println("Error: Failed to get a frame from the camera");
                     continue;
                 }
             } // else use the image from disk that we loaded above
-
-            if (use_GUI)
-                Imgproc.resize( frame, frame, visionParams.sz );
+            if(use_GUI)
+            //Resize the frame
+            Imgproc.resize( frame, frame, visionParams.sz );
 
             // Process the frame!
+            //Log when the pipeline starts
             long pipelineStart = System.nanoTime();
+            //Process the frame
             VisionData visionData = Pipeline.process(frame, visionParams, use_GUI);
+            //Log when the pipeline stops
             long pipelineEnd = System.nanoTime();
-
+            //Selects the prefered target
             Pipeline.selectPreferredTarget(visionData, visionParams);
-
-            Mat outputTargetImg = frame.clone();
-            Pipeline.drawPreferredTarget(outputTargetImg, visionData);
-
+            //Creates the raw output image object
+            Mat rawOutputImg;
+            if (use_GUI) {
+            	//If use gui then draw the prefered target
+                //Sets the raw image to the frame
+            	rawOutputImg = frame.clone();
+                //Draws the preffered target
+                Pipeline.drawPreferredTarget(rawOutputImg, visionData);
+            } else {
+                //Sets the raw image to the frame
+            	rawOutputImg = frame;
+            }
+            //Sends the data to the vision table
```

Comment:
Hmm, this is actually a code change. You or I will need to go back through the commit history to figure out which is the correct version.

Reply:
This is might be related to the `binMask` change below, but I don't remember off the top of my head.